src/test/java/org/prebid/server/bidder/outbrain/OutbrainBidderTest.java

```diff
@@ -71,6 +71,46 @@ public void makeHttpRequestsShouldCreateOneSingleRequestWithAllImps() {
                 .hasSize(2);
     }
 
+    @Test
+    public void makeHttpRequestsShouldModifyImpTagIdOnlyIfItPresentInExt() {
+        // given
+        final Imp firstImp = Imp.builder()
+                .id("123")
+                .tagid("123")
+                .ext(mapper.valueToTree(ExtPrebid.of(null, ExtImpOutbrain.of(
+                        ExtImpOutbrainPublisher.of("testId", "testName", "testDomain"),
+                        null, singletonList("testBcat"), singletonList("testBadv")))))
+                .build();
+
+        final Imp secondImp = Imp.builder()
```

Comment:
i'd say to test `tagid` we need only one imp.
Let's keep tests as small as possible.